barebones/ee_printf.c

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-#include <coremark.h>
```

Comment:
I think you can use -L to add to your library path rather than using the relative path

Reply:
sure thing, added